cmd/cloak/clientgen.go

```diff
@@ -75,6 +82,19 @@ func ClientGen(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getLang(cmd *cobra.Command) (string, error) {
+	pkg, err := cmd.Flags().GetString("lang")
+	if err != nil {
+		return "", err
+	}
+
+	// If a package name was provided as a flag, use it
+	if pkg != "" {
+		return pkg, nil
+	}
+	return "", nil
```

Comment:
That's basically the same, you can still return `pkg` if it's `""` so you return `""` without doing an if statement?

Reply:
fixed